in_toto/util.py

```diff
@@ -103,12 +87,32 @@ def import_rsa_key_from_file(filepath, password=None):
   return rsa_key
 
 
-def import_rsa_public_keys_from_files_as_dict(filepaths):
+def import_public_keys_from_files_as_dict(filepaths, key_types=None):
   """Takes a list of filepaths to RSA public keys and returns them as a
   dictionary conformant with securesystemslib.formats.KEYDICT_SCHEMA."""
+  # are key_types needed?
```

Comment:
Good point.  In the future, we'll revise the SSLib functions to better handle key files.